src/stcal/dark_current/dark_class.py

```diff
@@ -0,0 +1,87 @@
+import numpy as np
+
+
+class DarkData:
+    """
+    Class removing data model dependencies.
```

Comment:
I think we need a one line sentence summarizing what the class represents, not why it was created.

Reply:
I changed this comment.